ONSdigital/design-system
# Fix custom attributes for mutually exclusive checkboxes

### What is the context of this PR?
Fixes: #949 

### How to review
- Check that custom attributes can now be added to mutually exclusive checkboxes
- Check that checkboxes still work correctly
